File: app/src/main/java/com/libandexample/tutorials/simplifiedcoding/androidnotificationtutorial/NotificationHelper.java
package com.libandexample.tutorials.simplifiedcoding.androidnotificationtutorial;

import android.content.Context;
import androidx.core.app.NotificationCompat;
import androidx.core.app.NotificationManagerCompat;

import com.libandexample.R;

public class NotificationHelper {

    public static void displayNotification(Context context, String title, String body) {

        NotificationCompat.Builder mBuilder =
                new NotificationCompat.Builder(context, AndroidNotificationMainActivity.CHANNEL_ID)
                        .setSmallIcon(R.drawable.ic_notification)
                        .setContentTitle(title)
                        .setContentText(body)
                        .setPriority(NotificationCompat.PRIORITY_HIGH);

        NotificationManagerCompat mNotificationMgr = NotificationManagerCompat.from(context);
        mNotificationMgr.notify(1, mBuilder.build());

    }

}

File: app/src/main/java/com/libandexample/tutorials/simplifiedcoding/androidnotificationtutorial/NotificationHelper.kt
package com.libandexample.tutorials.simplifiedcoding.androidnotificationtutorial

import android.content.Context
import androidx.core.app.NotificationCompat
import androidx.core.app.NotificationManagerCompat

import com.libandexample.R

internal object NotificationHelper {

    fun displayNotification(context: Context, title: String, body: String) {

        val mBuilder = NotificationCompat.Builder(context, AndroidNotificationMainActivity.CHANNEL_ID)
                .setSmallIcon(R.drawable.ic_notification)
                .setContentTitle(title)
                .setContentText(body)
                .setPriority(NotificationCompat.PRIORITY_HIGH)

        val mNotificationMgr = NotificationManagerCompat.from(context)
        mNotificationMgr.notify(1, mBuilder.build())

    }

}
